contracts/erc20-bridge-sampler/test/erc20-bridge-sampler.ts

```diff
@@ -716,6 +718,93 @@ blockchainTests('erc20-bridge-sampler', env => {
         });
     });
 
+    blockchainTests.resets('getLiquidityProviderFromRegistry', () => {
```

Comment:
Nit: All the test cases here are constant, so no chain state to reset. This can just be a regular `describe()` block.